training/general/src/environments/sklearn-1.1/context/intel-extension/conda_dependencies.yaml

```
@@ -0,0 +1,22 @@
+name: sklearn-1.1
+channels:
+- conda-forge
+- anaconda
+dependencies:
+- python=3.9
+- pip=21.3.1
+- pandas~=1.3.0
+- scipy~=1.7.0
+- numpy~=1.21.0
+- scikit-learn-intelex~={{latest-sklearnex-version}}
```

This will pull the latest version from PyPI when this environment is built:
```suggestion
- scikit-learn-intelex~={{latest-pypi-version}}
```

---

I see. Thanks. Removed the `sed` instructions from README.md.